src/spawn.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  spawn as SpawnInternal,
+  SpawnOptionsWithStdioTuple,
+  StdioNull,
+  StdioPipe
+} from 'child_process'
+
+export type SpawnReturnArgs = {
+  exitCode: number | null
+}
+
+/**
+ * Simple wrapper around NodeJS's "child_process.spawn" function.
+ * Since we only use the exit code, we only expose that.
+ */
+export async function spawn(
+  command: string,
+  args: readonly string[],
+  options: SpawnOptionsWithStdioTuple<StdioPipe, StdioNull, StdioNull>
```

Comment:
We have two callers that both pass in `[undefined, 'inherit', 'inherit']` here. How about hard-coding it and dropping the `options` function parameter?

Reply:
Great suggestion, thanks!